media-sound/mp3gain/mp3gain-1.5.2.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="Lossless mp3 normalizer with statistical analysis"
+DESCRIPTION="MP3Gain analyzes and adjusts mp3 files so that they have the same \
+volume. MP3Gain does not just do peak normalization, as many normalizers do. \
+Instead, it does some statistical analysis (using the replaygain algorithm)to \
+determine how loud the file actually sounds to the human ear.
+MP3Gain can adjust the volume in a completely lossless way by modifying the mp3 \
+file directly, without decoding and re-encoding. This works with all mp3 \
```

Comment:
```diff
-MP3Gain can adjust the volume in a completely lossless way by modifying the mp3 \
+MP3Gain can adjust the volume in a completely lossless way by modifying the \
-file directly, without decoding and re-encoding. This works with all mp3 \
+mp3 file directly, without decoding and re-encoding. This works with all mp3 \
```
Line 6 was OK. Now it has is 81 chars...
(Yep, we count the backslash.)
(Sorry for being a little bit picky.)

(Everything else is OK :)